changes/unreleased/4990.HxHP2zqbTAcGzjapkgX3E5.toml

```diff
@@ -0,0 +1,5 @@
+internal = "Update configuration of actions/stale to use issue types instead of labels"
+
+[[pull_requests]]
+uid = "4890"
+author_uids = ["Bibo-Joshi"]
```

Comment:
does this not need a `closes_threads = []`?

Reply:
no, that's optional :)